src/EnergyPlus/EconomicLifeCycleCost.cc

```diff
@@ -1461,6 +1460,14 @@ namespace EconomicLifeCycleCost {
 				CashFlow( jCost ).yrAmount( kYear ) = annualCost;
 			}
 		}
+		// generate a warning if resource referenced was not used
+		for ( int nUsePriceEsc = 1; nUsePriceEsc <= numUsePriceEscalation; ++nUsePriceEsc ) {
+			int curResource = UsePriceEscalation( nUsePriceEsc ).resource - ResourceTypeInitialOffset;
+			if ( !resourceCostNotZero( curResource ) && DataGlobals::DoWeathSim ) {
+				ShowWarningError( "The resource referenced by LifeCycleCost:UsePriceEscalation= \"" + UsePriceEscalation( nUsePriceEsc ).name + "\" has no energy cost. " );
+				ShowContinueError( "... It is likely that the wrong resource is used. The resource should match the meter used in Utility:Tariff." );
+			}
+		}
```

Comment:
Or they just got some free stuff.  We should `ShowCongratulations` to them as well as the warning.

Reply:
:-)